csrankings-a.csv

```diff
@@ -4,6 +4,7 @@ A. Akbari Azirani,IUST,http://ce.iust.ac.ir/page.php?slct_pg_id=6537&sid=14&slc_
 A. Akbariazirani,IUST,http://ce.iust.ac.ir/page.php?slct_pg_id=6537&sid=14&slc_lang=en,pCil4_cAAAAJ
 A. Aldo Faisal,Imperial College London,https://www.imperial.ac.uk/people/a.faisal,WjHjbrwAAAAJ
 A. Antony Franklin,IIT Hyderabad,http://www.iith.ac.in/~antony/index.html,LVfqLuoAAAAJ
+A. Aminifar,Lund University,https://www.eit.lth.se/staff/amir.aminifar,4eSC97MAAAAJ

```

Comment:
Incorrect DBLP name. Should be "Amir Aminifar".

Reply:
Sorry, it should be fixed now.